### DISCUSSION TITLE
Which exact permission is required to create initial access tokens?

### DISCUSSION THREAD

**Discussion by martin31821** [keycloak/keycloak#20281] [Q&A] [answered]
Hi all,

I'm asking here because I couldn't find any other documentation source for my exact problem.
We're running an app where IATs are used across several realms quite extensively, and we want to dynamically create IATs.

My initial idea was to create an OpenID client in the master realm and give it the appropriate permission to issue IATs, which brings me to my question: 
Which permission do I need to issue IATs?

**Comment by abstractj** [comment, marked as answer]
@martin31821 I believe you need the "manage-authorization" permission for the realm in which you want to create the token.

**Comment by martin31821** [comment]
Is there a system wide version of this, I. E. Can I assign it for all realms in the system?
________________________________
